src/main/java/lotto/service/BuyLotto.java

```diff
@@ -0,0 +1,46 @@
+package lotto.service;
+
+import lotto.domain.Lotto;
+import lotto.domain.Lottos;
+import lotto.domain.Money;
+
+public class BuyLotto {
+    private final int BUY_AMOUNT_UNIT = 1000;
+    private final String BUY_AMOUNT_UNIT_CHECK_TEXT = "금액은 " + BUY_AMOUNT_UNIT + "원 단위로 입력하십시오.";
+    private final int buyAmount;
+    private final int count;
+    private Lottos lottos = new Lottos();
+
+    public BuyLotto(Money money) {
+        validateCheck(money);
+        this.buyAmount = money.getBuyAmount();
+        this.count = buyLottoCount(money);
+    }
+
+    public int getbuyAmount() {
+        return this.buyAmount;
+    }
+
+    public int getCount() {
+        return this.count;
+    }
+
+    private void validateCheck(Money money) {
+        if (money.getBuyAmount() % BUY_AMOUNT_UNIT != 0
+                || money.getBuyAmount() <= 0) {
```

Comment:
```suggestion
                || money.lessThan(0)) {
```


객체를 잘 만들어주셨습니다!
그렇다면, 객체를 좀 더 능동적으로 활용하는 방향을 고민해보면 좋을듯해요!

객체에게 비교하도록 메시지를 전달하는 패턴을 고민해보시면 좋겠어요!